sdk/appconfiguration/azure-appconfiguration-provider/tests/recordings/test_async_provider.pyTestAppConfigurationClientAADtest_add_configuration_setting.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't find the test in your latest change, you can remove the odd recording or re-record. It should have some content with the new appconfiguration release.

Reply:
They must have gotten in while trying to test out that async issue. I have removed them.